### PR TITLE
Open Source : 6.17 explain everything related to tensorflow

### LINKED ISSUE #3328: Open Source : 6.17 explain everything related to tensorflow
### Description

expalin everything related to tensorflow

### Domain

Open Source

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Adi7171**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #3328

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Open #Source

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Adi7171** [comment]
@Aryamanz29 How is it

File: Open_Source/6.Popular_Projects/6.18_Tensorflow.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
...e7YTmpHIZI3Zo4l8Ppc+0h1K3/+Ke66uBBcQVCO+Z7KauUhPNQ2EbqcdFDT8qF00t81TWVEchqF1bEM9Rxc1MvEmjTK6e6a7nduhyGSx52RWnJj1Ho857q/8fZlKiCAJXuWTyvJSDboV6mpJlgiMrB41whvl8od0zTQ54RbH+eAsrKyUICdrZu+p0b1Rl6NvN3fc0HSykazOoiyVxaGnHcZMaPtPgawhhWRHeU0/WTdwh0y3SbG5mZ/LZ7c7iSyZk8VDLY1mkTLjfKEHHQV6FfVc7mux1L21EiSyuLLpaHFMrCqQxQ8ReKKXwTTU6T3TmQxjqePMiqziAK8nzTQxMPHL9Cfa5nucU+re+ih7eRTKIoI1smjSozUqT47S5S+cR16Y3IxlLXaSVUsTrayhYvk3Dl1Aw5T6lMzmitLJy8omLWK0RhbFmk6XFmM0699ePdq6ULdwJ1vLGqncanOWTBQu+9aaaEi6bapX9mWUTT5mqeHyZ2tkMVFMq30Oxe38dskGsC1lRUEtWxTwui2wH0qeE4V+5janw8fVKo9UFgX1LPgYVmUNfd+3VfdN65JZTEPTPce6LWUJXsYlE3Cse5QtlOZCXrOTTZcCviOqQior5t1K8j2bJwirssa3MljYFDt90dpb2gfqycp1W1ndRBDBc4oebtzfklUMLWGcyux1RE7WjLbDztysCMJAT/GrsiK77BTGDE1WPA5rUicNeb/UFdvL4gWtNOsO3mvZq+a8LTfHsd70VGSvI/J7w7HiKa47Hrfn1FiubfGitCbbo+GIY5ZZYPFjfEXZuslueGtZeg8up2En9tiV7Z9sn6cJoafGWmX2OmLpEU0z0CsoXhi4vIAvDPBzborUqwf70GYodRL/65jN5dayxEQL1kuR9N0ST5JJb/Kc3P6gfLrKlcls1qGtsX4DoxZ6OPJbrGQhdGv3daKnXPOaRk3sR9FC2WxTE/x9ypfJopPb5TJ70lV2udEOTFYpsxeJXJTtTW2VXlkFxua9Z3LWm3vetGm3F/x+NKkqvx81AW3YXniet2jnHl6OmlPPm3eTme0ql8+8ZF0tMwlEUUx3m9+EubEWZ7mHdGVlFg4AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwX/gazIc4dkxJGbwAAAABJRU5ErkJggg==" alt="Sublime's custom image"/>
```
Comment (Aryamanz29) [inline Open_Source/6.Popular_Projects/6.18_Tensorflow.md:7]: Omg 😱, @Adi7171  Remove this Image link from `doc` or use a URL shortener, We don't need our whole covered with a single link

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

File: Open_Source/6.Popular_Projects/6.18_Tensorflow.md (hunk 1 above)
Comment (Aryamanz29) [reply Open_Source/6.Popular_Projects/6.18_Tensorflow.md:7]: ## Also image not completely fetched :arrow_down: 

Screenshot from 2021-10-04 21-23-17

**Comment by Adi7171** [comment]
@Aryamanz29 Does hactoberfest tag mean that this pr will be counted for hacktoberfest

File: Open_Source/6.Popular_Projects/6.18_Tensorflow.md (hunk 1 above)
Comment (Adi7171) [reply Open_Source/6.Popular_Projects/6.18_Tensorflow.md]: done

**Comment by A-GHOSH-dev** [comment]
Can this be assigned to me?

**Review by Aryamanz29 — APPROVED** [review]
Awesome work 
Thanks for contributing @Adi7171 🎉🎉

**Comment by Aryamanz29** [comment]
> Can this be assigned to me?

@A-GHOSH-dev  I'm creating some new issues today, You can check out our [domain milestone](https://github.com/girlscript/winter-of-contributing/milestone/2) for issue assignment
Thanks

**Pull request merged**